apps/mull-api/src/app/event/event.resolver.ts

```diff
@@ -34,6 +34,14 @@ export class EventResolver {
     return this.eventService.findJoinedEvents(id);
   }
 
+  @Query(/* istanbul ignore next */ () => Boolean)
+  async participatingEvent(
+    @Args('eventId', { type: /* istanbul ignore next */ () => Int }) eventId: number,
+    @Args('userId', { type: /* istanbul ignore next */ () => Int }) userId: number
+  ) {
+    return !!(await this.eventService.findJoinedEvent(eventId, userId));
```

Comment:
`findJoinedEvent` returns a promise for an event. Instead of doing the `!!` thing you can simply change what `findJoinedEvent` returns to be a boolean. when successful